src/arith/rewrite_simplify.h

```diff
@@ -39,6 +40,36 @@ namespace arith {
 
 using namespace tir;
 
+struct RewriteSimplifierStatsNode : Object {
+  int nodes_visited{0};
+  int constraints_entered{0};
+  int rewrites_attempted{0};
```

Comment:
use int64_t

Reply:
Thank you, updated here and in the `SetMaximumRewriteSteps` method.